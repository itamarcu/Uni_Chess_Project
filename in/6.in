game_mode 2
start
move <2,D> to <4,D>
move <2,D> to <4,D>
move <8,G> to <6,F>
move <1,B> to <2,D>
move <7,E> to <5,E>
move <4,D> to <5,E>
move <6,F> to <4,G>
move <2,H> to <3,H>
move <4,G> to <3,E>
move <2,C> to <4,C>
move <3,E> to <1,D>
quit
