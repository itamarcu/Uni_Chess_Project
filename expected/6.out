 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m _ m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
The specified position does not contain your piece
Enter your move (black player):
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m _ m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n m m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ M _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n m m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n m m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ _ _ _ _ N _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n m m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ _ _ _ _ N _ |
3| _ _ _ _ _ _ _ m |
2| m m m n m m m _ |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ N _ _ m |
2| m m m n m m m _ |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ m _ _ _ _ _ |
3| _ _ _ _ N _ _ m |
2| m m _ n m m m _ |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ m _ _ _ _ _ |
3| _ _ _ _ _ _ _ m |
2| m m _ n m m m _ |
1| r _ b N k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Exiting...
